components/providers/ContentAccordion.vue

```diff
@@ -27,7 +27,7 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component, Prop } from 'vue-property-decorator'
-import { AccordionLayoutProps } from '~/components/overview/AccordionLayout.vue'
+import { AccordionLayoutProps } from './AccordionLayout.vue'
```

Comment:
```suggestion
import { AccordionLayoutProps } from '~/components/providers/AccordionLayout.vue'
```

Reply:
Updated in this commit https://github.com/Qiskit/qiskit.org/pull/2964/commits/85154b31a7e84597e6e5e2c5d6454f5c9ae45e31